tutorials/test_status/regex_on_invalids.yml

```diff
@@ -0,0 +1,22 @@
+build specs:
+  regex_on_invalid_fname:
+    type: script
+    executor: generic.local.bash
+    description: Test regex on an invalid file name
+    run: |
+      echo "Hello" > hello.txt
+    status:
+      file_regex:
+      - file: /fox.txt
+      - exp: '^(Hello)$'
+
+regex_where_validfname_invaildregex:
+  type: script
+  executor: generic.local.bash
+  description: Test regex on a valid file where regex is invaild
+  run: |
+    echo "Jump" > jump.txt
```

Comment:
please put this on a single line `run: echo "Jump" > jump.txt`

If its multi line then do what you did